CSharp/BatchExplorer/Helpers/BasicDataProvider.cs

```diff
@@ -1,137 +1,151 @@
-﻿//Copyright (c) Microsoft Corporation
```

Comment:
Sure. But just FYI, this is because VS said these weren't standard Windows CRLF and VS wanted to convert them, so I let it.